include/verilated_trace_threaded.h

```diff
@@ -0,0 +1,103 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//=============================================================================
+//
+// THIS MODULE IS PUBLICLY LICENSED
+//
+// Copyright 2001-2020 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//=============================================================================
+///
+/// \file
+/// \brief Threaded tracing internals
+///
+//=============================================================================
+// SPDIFF_OFF
+
+#ifndef _VERILATED_TRACE_THREADED_H_
+#define _VERILATED_TRACE_THREADED_H_ 1
+
+#include <deque>
```

Comment:
Please always include a verilated_something so we get the verilatedos.h stuff which fixes some behaviors.

```suggestion
#include "verilated_threads.h"
#include <deque>
```

Reply:
This file is only every included in `verilated_trace.h` so it does get the `verialedos.h` indirectly. I was expecting it to be bigger when I started, but now I'm leaning to just inline this in `verilated_trace.h` under `VL_TRACE_THREADED`.

Reply:
Inlining seems ideal. BTW we try to follow the (Google etc) recommendation that all header files should be separately compilable, even when it might not be needed.  Avoids problems down the road when things refactor.